docs/en/other-authenticators.md

```diff
@@ -0,0 +1,129 @@
+# Integrating with other authenticators
+
+**This version relates to SilverStripe 4.x configuration.**
+
+If your project uses a non-standard authentication module, such as silverstripe/ldap, you will
+need to implement some customisations to connect the modules together. The following notes should serve as a guide
+for parts of the code to be aware of, and things to do in order to achieve this.
+
+For the purposes of comparisons in this document, we will use silverstripe/ldap's authenticator.
+
+## Concepts
+
+### Configuration
+
+The MFA module enables and sets itself as the `default` authenticator with config (see `config.yml`). You can leave this
+in place, but change the injected class to a custom class in your project with Injector configuration. See further
+down for a hypothetical example.
+
+### Authenticator
+
+The Authenticator entrypoint class in the MFA module is `SilverStripe\MFA\Authenticator\MemberAuthenticator`. This
+class extends the default `SilverStripe\Security\MemberAuthenticator` class in order to override the default login
+form with `SilverStripe\MFA\Authenticator\LoginForm`, and the change password handler with
+`SilverStripe\MFA\Authenticator\ChangePasswordHandler`.
+
+silverstripe/ldap does the same thing - it also configures itself to override the `default` authenticator. Since the
+MFA replacement for the default authenticator has MFA logic added to it, and LDAP has the same with LDAP logic added, 
+you will need to re-implement it in a way that is applied to both MFA and LDAP simultaneously.
```

Comment:
```suggestion
you will need to reimplement it so that both MFA and LDAP apply their logic together.
```

This sentence felt a little weird to me - just a suggestion (although I also removed the hyphen from "reimplement")